src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -253,6 +253,22 @@ public static string CleanTitle(string title)
             return title;
         }
 
+        public static string TitleThe(string title)
+        {
+            string[] prefixes = { "The ", "An ", "A " };
+            foreach (string prefix in prefixes)
+            {
+                int prefix_length = prefix.Length;
+                if (prefix.ToLower() == title.Substring(0, prefix_length).ToLower())
```

Comment:
Of course `string.StartsWith` is a better option, I would have used it had I wrote the code from scratch. I copied and pasted this code from Radarr. No excuse for not reviewing the code correctly myself, I should have put more than the minimum amount of effort in, my apologies.